test/harry/main/org/apache/cassandra/harry/dsl/HistoryBuilderHelper.java

```diff
@@ -113,19 +118,24 @@ public static void deleteRandomColumns(SchemaSpec schema, int partitionIdx, int
      * @param population - expected population / number of possible values for a given column
      * @return a list of relations
      */
-    public static List<SingleOperationBuilder.IdxRelation> generateValueRelations(EntropySource rng, int numColumns, Function<Integer, Integer> population)
+    public static List<SingleOperationBuilder.IdxRelation> generateValueRelations(EntropySource rng, int numColumns, Function<Integer, Integer> population, Set<Integer> eqOnlyColumns)
```

Comment:
left this comment at the usage, but `IntPredicate` might be better for this api?  `eqOnlyColumns.contains(column)` could become `eqOnly.test(column)`